src/ChipInput.js

```diff
@@ -161,9 +206,9 @@ class ChipInput extends React.Component {
     if (this.state.focusedChip != null) {
       this.setState({ focusedChip: null })
     }
-  }
+  };
```

Comment:
I'm pretty sure that this won't lint. We use standard linter (`npm run lint`)

Reply:
Apologies for the added semis and other formatting - my IDE was not playing nicely with your formatting standards, and I wasn't being careful enough to notice these changes. I'll get them cleaned out!

Reply:
Don't worry. I really like this PR, no matter the formatting. :wink: `npm run lint -- --fix` can probably fix most of the issues.